media/server/ipc/include/SetLogLevelsService.h

```diff
@@ -20,11 +20,13 @@
 #ifndef FIREBOLT_RIALTO_SERVER_IPC_SET_LOG_LEVELS_SERVICE_H_
 #define FIREBOLT_RIALTO_SERVER_IPC_SET_LOG_LEVELS_SERVICE_H_
 
-#include "RialtoServerLogging.h"
-#include <IIpcServer.h>
 #include <memory>
+#include <mutex>
 #include <set>
 
+#include "IIpcServer.h"
```

Comment:
All headers are required except "RialtoServerLogging.h" can be replaced by "RialtoLogging.h" since the code beneath just requires the definition of RIALTO_DEBUG_LEVEL